### ISSUE TITLE
email validation

### ISSUE THREAD

**Issue by dexterns88** [final-form/react-final-form-html5-validation#26]
### What is the current behavior?
Valid input when enter: abc.def@mail

### What is the expected behavior?
Show error on: abc.def@mail - invalid email

### Other information

you can check issue here https://codesandbox.io/s/14r018yjp4
with type mail like abc.def@mail into field and no error

**Comment by rishabhsrao** [comment]
I don’t think this is an issue because `abc.def@mail` matches the pattern of a valid email address. As another example, `root@localhost` is a perfectly valid email address.

For more info, see https://stackoverflow.com/questions/20573488/why-does-html5-form-validation-allow-emails-without-a-dot and https://stackoverflow.com/questions/14373433/how-does-html5-input-type-email-works-without-top-level-domain-name
